src/core/Microsoft.PowerToys.Settings.UI.Lib/Utilities/Helper.cs

```diff
@@ -78,5 +80,40 @@ public static string GetKeyName(uint key)
         {
             return layoutMap.GetKeyName(key);
         }
+
+        public static string GetProductVersion()
+        {
+            return interop.CommonManaged.GetProductVersion();
+        }
+
+        public static int CompareVersions(string version1, string version2)
+        {
+            try
+            {
+                var v1 = version1.Substring(1).Split('.').Select(int.Parse).ToArray();

```

Comment:
what is v1 and what is v2? comments could be helpful as to what we are comparing or evaluating

Reply:
Version1 and version2 are two string denoting versions. v1 and v2 are the version numbers in array form.

Reply:
It seems to me that this method just checks if two versions are equal and in that case it should be renamed to what it does and return a boolean value.

Reply:
I returns < 0 if version1 is smaller, 0 if they are equal and > 0 if version2 is larger. Its the standard convention when comparing stuff as far as I know.